src/test/robot/api/storage/data/create_and_read_events.js

```diff
@@ -0,0 +1,45 @@
+PreconditionLoad
+	.config({
+		"item" : {
+			"type" : "data",
+			"output" : {
+				"path" : "test"
```

Comment:
use single value specified by the environment variable for both write and read
see other fun tests for example